go.mod

```diff
@@ -21,7 +21,7 @@ module github.com/apache/yunikorn-k8shim
 go 1.20
 
 require (
-	github.com/apache/yunikorn-core v0.0.0-20240103094035-ba62c5db9f61
+	github.com/apache/yunikorn-core v0.0.0-20240105094327-77e19f6aca27
```

Comment:
Have to update core version to use the latest api:
- ws/v1/partition/default/applications/completed